modules/programs/firefox/mkFirefoxModule.nix

```
@@ -341,6 +340,13 @@ in {
       };
     });
 
+    profileVersion = mkOption {
+      inherit visible;
```

```suggestion
      internal = true;
```

It would make sense to mark this as internal.